### PR TITLE
Add more e2e

### PR DESCRIPTION
**Pull request by surangap**
Add more e2e

### CODE REVIEW

File: e2e/test/Doughnuts.test.ts [hunk 1]
```diff
@@ -999,4 +1000,506 @@ describe("Doughnuts", () => {
...
+          console.log(err);
+        });
+
+      // balances after
+      const alice_balance_after = ((await api.query.system.account(alice.address)).toJSON() as any)?.data.free ?? 0;
+      const receiver_balance_after =
+        ((await api.query.system.account(receiverAddress)).toJSON() as any)?.data.free ?? 0;
+
+      expect(Math.abs(receiver_balance_before - receiver_balance_after)).to.be.equal(0);
+      expect(alice_balance_before - alice_balance_after).equal(0);
```
Comment (JasonTulp) [inline e2e/test/Doughnuts.test.ts:1139]: nit, expect(alice_balance_before).equal(alice_balance_after);

File: e2e/test/Doughnuts.test.ts [hunk 2]
```diff
@@ -999,4 +1000,506 @@ describe("Doughnuts", () => {
...
+      // balances after
+      const alice_balance_after = ((await api.query.system.account(alice.address)).toJSON() as any)?.data.free ?? 0;
+      const receiver_balance_after =
+        ((await api.query.system.account(receiverAddress)).toJSON() as any)?.data.free ?? 0;
+
+      expect(Math.abs(receiver_balance_before - receiver_balance_after)).to.be.equal(transferAmountLimit);
+      expect(alice_balance_before - alice_balance_after).equal(transferAmountLimit);
+    }
+
+    await sleep(4000); // TODO: remove this
```
Comment (JasonTulp) [inline e2e/test/Doughnuts.test.ts:1321]: TODO

File: e2e/test/Doughnuts.test.ts [hunk 3]
```diff
@@ -999,4 +1000,506 @@ describe("Doughnuts", () => {
...
+      // Execute the transact call with.send
+      await api.tx.doughnut
+        .transact(call, doughnutHex, nonce, holderSig)
+        .send()
+        .catch((err: any) => {
+          console.log(err);
+        });
+    }
+
+    await sleep(4000); // TODO: remove this
```
Comment (JasonTulp) [inline e2e/test/Doughnuts.test.ts:1481]: TODO

**Review by JasonTulp — APPROVED** [review]
Nice! Good to have more tests

File: e2e/test/Doughnuts.test.ts (hunk 2 above)
Comment (surangap) [reply e2e/test/Doughnuts.test.ts:1321]: got some mempool error related to having the same priority. should remove it once we have priority in placed

**Pull request merged**